.develop.env

```
@@ -1,8 +1,9 @@
-SERVICE_NAME=tesoreria_mid
-API_BASE_DIR=github.com/udistrital
-API_NAME=tesoreria_mid
-API_PORT=8081
-RUN_MODE=dev
+export SERVICE_NAME=tesoreria_mid
+export API_BASE_DIR=github.com/udistrital
+export API_NAME=tesoreria_mid
+export API_PORT=8081
+export RUN_MODE=dev
 # APIs
-AVANCES_CRUD_URL=http://172.19.0.3:8080/v1/
-SOLICITUDES_CRUD_URL=http://pruebasapi2.intranetoas.udistrital.edu.co:8117/v1/
\ No newline at end of file
+export AVANCES_CRUD_URL=http://pruebasapi2.intranetoas.udistrital.edu.co:8212/v1/
+export SOLICITUDES_CRUD_URL=http://pruebasapi2.intranetoas.udistrital.edu.co:8117/v1/
+export TERCEROS_CRUD_URL=http://pruebasapi.intranetoas.udistrital.edu.co:8121/v1/
```

(En general) Dejar vacía la última línea de cualquier archivo de texto plano seguido por Git
```suggestion
export TERCEROS_CRUD_URL=http://pruebasapi.intranetoas.udistrital.edu.co:8121/v1/

```